features/aanschrijfwijze-persoon-zonder-partner-met-adellijke-titel-predicaat.feature

```diff
@@ -0,0 +1,356 @@
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | aanschrijfwijze                 |
+      Dan heeft de response een persoon met de volgende 'adressering' gegevens
+      | naam                          | waarde                   |
+      | aanschrijfwijze.naam          | F. Groenen               |
+      | aanschrijfwijze.aanspreekvorm | De hoogwelgeboren vrouwe |
+
+  Rule: voorvoegsel wordt gebruikt zoals het is geregistreerd
+
+    Abstract Scenario: persoon met voorvoegsel, heeft geslacht "<geslacht>" en een adellijke titel
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                        | waarde     |
+      | burgerservicenummer         | 999992934  |
+      | geslachtsaanduiding (04.10) | <geslacht> |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                                 | waarde            |
+      | voorvoegsel (02.30)                  | <voorvoegsel>     |
+      | geslachtsnaam (02.40)                | Aedel             |
+      | voornamen (02.10)                    | <voornamen>       |
+      | adellijke titel of predicaat (02.20) | <adellijke titel> |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | aanschrijfwijze                 |
+      Dan heeft de response een persoon met de volgende 'adressering' gegevens
+      | naam                          | waarde                                      |
+      | aanschrijfwijze.naam          | F. <titel omschrijving> <voorvoegsel> Aedel |
+      | aanschrijfwijze.aanspreekvorm | <aanspreekvorm>                             |
+
+      Voorbeelden:
+      | geslacht | voornamen | voorvoegsel | adellijke titel | titel omschrijving | aanspreekvorm            |
+      | M        | Franklin  | Van Den     | R               | ridder             | De hoogwelgeboren heer   |
+      | M        | Franklin  | van den     | B               | baron              | De hoogwelgeboren heer   |
+      | M        | Franklin  | van         | GI              | graaf              | De hooggeboren heer      |
+      | V        | Francisca | Van         | B               | barones            | De hoogwelgeboren vrouwe |
+      | M        | Franklin  | van den     | P               | prins              | De hoogheid              |
+      | V        | Francisca | van den     | PS              | prinses            | De hoogheid              |
+
+  Rule: de eerste naamcomponent in de naam in aanschrijfwijze begint met een hoofdletter
+
+    Abstract Scenario: persoon heeft een naamketen (alleen geslachtsnaam) en een adellijke titel
```

Comment:
Wordt buitenlandse adel ook in de BRP geregistreerd? Of alleen Nederlandse adel? Wanneer alleen NL adellijke titels worden opgenomen kan dit scenario kan niet voorkomen. 
Bij nader inzien: maakt ook eigenlijk niet uit. Hoewel hypothetisch: zo laten, en zo implementeren!

Reply:
> Wordt buitenlandse adel ook in de BRP geregistreerd? Of alleen Nederlandse adel? Wanneer alleen NL adellijke titels worden opgenomen kan dit scenario kan niet voorkomen. Bij nader inzien: maakt ook eigenlijk niet uit. Hoewel hypothetisch: zo laten, en zo implementeren!

Naamswijziging van iemand met een adellijke titel/predicaat, omdat hij de islam gaat beleiden. V.b. hiervan, Cat Stevens die nu Yusuf Islam heet